person.rb

```diff
@@ -11,6 +11,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Kindly make sure to add the `rentals` to `attr_accessor` so it can be accessible

Reply:
Well resolved 👍🏽

Reply:
Well resolved 👍🏽